component/main.jsonnet

```
@@ -5,6 +5,79 @@ local inv = kap.inventory();
 // The hiera parameters for the component
 local params = inv.parameters.eventrouter;
 
+local serviceaccount = kube.ServiceAccount('eventrouter') {
+  metadata+: {
+    namespace: params.namespace,
+  },
+};
+
+local clusterrole = kube.ClusterRole('eventrouter') {
```

To avoid any collisions
```suggestion
local clusterrole = kube.ClusterRole('syn-eventrouter') {
```